src/classes/Standardizer/plugins/Google/getters/getNotes.ts

```diff
@@ -0,0 +1,32 @@
+import Google from '../Google'
+import { Message } from '../../../../../types/schemas'
+
+interface GoogleNotes {
+  color: string,
+  isTrashed: boolean,
+  isPinned: boolean,
+  isArchived: boolean,
+  textContent?: string,
+  title?: string,
+  userEditedTimeStampUsec: number
+}
+
+Google.prototype.getNotes = async function getNotes(options) {
+  const NOTES_FILES = await this.parser.listFiles('Takeout/Keep/', { extensionWhitelist: ['json'] })
+
+  const notes = await Promise.all(NOTES_FILES.map(async file => {
+    const note = await this.parser.parseAsJSON<GoogleNotes>(file, options?.parsingOptions)
+    return {
+      sender: 'Note to myself',
+      receiver: 'Note to myself',
+      title: note.title ?? 'No title provided',
+      content: note.textContent ?? 'No content provided',
+      creationDate: new Date(file.split('Takeout/Keep/')[1].replaceAll('_', ':').split('.json')[0]),
+    } as Message
```

Comment:
Need to rework this with a dedicated `Note` type

Reply:
fixed